portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,31 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 700px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.font {
+  font-family: Optima, Arial, Helvetica, sans-serif;
+}
+
+@media only screen and (max-width: 700px) {
+  p {
+    font-size: 16px;
+  }
+
+  img {
+    max-width: 25%;
```

Comment:
Have you tested this on the different screen sizes? I'm partially guessing at your intentions so I could be wrong about what you're trying to accomplish, but it looks to me like images will be a smaller percentage of the screen size when the screen is smaller right now, when I'd generally expect to see the opposite.

Reply:
I was mainly trying to avoid forcing the user to scroll horizontally to see an image that takes up an area greater than the page. That 25% is 25% of the div I set with fixed width of 700. If it's still set at 100% then it'll start going off the page as the width of the window decreases below 700. 
I decided to change the max-width to 50% for that screen size, so now it can get to 350 px without requiring horizontal scrolling to see the image.

I'm not too sure what the best option is for this, but I am open to changing it in the future.